app/config.ts

```diff
@@ -26,6 +26,7 @@ const fallbackConfig: Config = {
   MAP_CENTER: process.env.MAP_CENTER || '10.7522, 63.9139',
   MAP_ZOOM: parseInt(process.env.MAP_ZOOM || '4', 10),
   MAP_MAX_ZOOM: parseInt(process.env.MAP_MAX_ZOOM || '13', 10),
+  EMAIL: process.env.EMAIL,
```

Comment:
I don't know about the rest of you, but I feel we have been a bit inconsistent on what we put in the config vs what we put in the translations. We discussed this at one point, and I think we decided to put URLs, emails etc in the translation file. I also believe that is easier to maintain than the config (although both is hard 😓). In my opinion, we could at least move `EMAIL`, `TWITTER` (and possibly `ZIP_PLACEHOLDER`) to the translations. If we e.g. use ```__(`kontakt@bustbyte.no`)``` when rendering the email, it would automatically fallback to kontakt@bustbyte.no if it is not overriden. Thoughts on this?

Reply:
I would support the idea, but- AFAIK minority of the domains have email system configured for the domain. That's why I made it optional and only enable when explicitely enabled in the config. In this case I don't think it's better to move it to the translation files.

Reply:
Yeah I get the point. But do you think that it is better to don't display a contact email at all if it is not set? We have gotten a few questions and comments from different countries on our email, so it could be nice to have a fallback?

Reply:
Ah, I didn't realize that, it makes sense to have the fallback email :+1: 
So, should I move it to the translations then?

Reply:
Yeah or you could also do `EMAIL: process.env.EMAIL || 'kontakt@bustbyte.no'`. I think I would prefer translations, but you can use the config if it is easier :)

Reply:
Okay. According to these info, I will use the default value here and:
1. Keep `EMAIL` in the config to have it next to the `TWITTER`
2. Move both `EMAIL` and `TWITTER` in another PR to the translations (I guess moving the TW would require much more effort due to the testing of all other parts where TW is actually used)

Reply:
change commited

Reply:
Not sure if it's worth to add the option to `config.example.json` yet, not to confuse people if it's gonna move out soon.